apps/epic-web/src/components/calculations/calculate-totals.ts

```diff
@@ -1,56 +1,62 @@
-import {SimplifiedCharge, SimplifiedRefund} from 'lib/transactions'
+import {CombinedBalanceTransaction} from 'lib/transactions'
 
-type ProductGroup = {
+export type ProductGroup = {
```

Comment:
it might make sense to move these types up, maybe there's a "lib" here so we have an "sdk" for report building logic?